xtask/src/util.rs

```diff
@@ -68,6 +72,30 @@ pub fn get_matrix() -> &'static Vec<Matrix> {
         .unwrap()
 }
 
+pub fn format_repo(registry: &str, repository: &str) -> String {
+    let mut output = String::new();
+    if !repository.is_empty() {
+        output = repository.to_string();
+    }
+    if !registry.is_empty() {
+        output = format!("{registry}/{output}");
+    }
+
+    output
+}
+
+pub fn pull_image(engine: &Path, image: &str, verbose: bool) -> cross::Result<()> {
+    let mut command = Command::new(engine);
+    command.arg("pull");
+    command.arg(image);
+    if !verbose {
+        // capture output to avoid polluting table
+        command.stdout(Stdio::null());
+        command.stderr(Stdio::null());
+    }
+    command.run(verbose, false).map_err(Into::into)
```

Comment:
I know this is old but I think this should be changed to

```suggestion
    let out = command.run_and_get_output(verbose)?;
    command.status_result(verbose, out.status, Some(&out))?;
    Ok(())
```